app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js

```diff
@@ -254,14 +254,20 @@ class MigrationsCompletedList extends React.Component {
                             {elapsedTime}
                           </ListView.InfoItem>,
                           migrationScheduled &&
-                            !staleMigrationSchedule && (
-                              <ListView.InfoItem key={plan.id + 1} style={{ textAlign: 'left' }}>
+                            !staleMigrationSchedule &&
+                            !migrationStarting && (
+                              <ListView.InfoItem key={`${plan.id}-scheduledTime`} style={{ textAlign: 'left' }}>
                                 <Icon type="fa" name="clock-o" />
                                 {__(`Migration scheduled`)}
                                 <br />
                                 {formatDateTime(migrationScheduled)}
                               </ListView.InfoItem>
-                            )
+                            ),
+                          migrationStarting && (
+                            <ListView.InfoItem key={`${plan.id}-starting`} style={{ textAlign: 'left' }}>
+                              {__(`Migration in progress`)}
```

Comment:
gettext, if I am not mistaken, supports only single quotes or double quotes.

backticks are not supported.

@mturley Can you change all backticks here and below to single quotes?

/cc @mzazrivec

Reply:
Whoops! Looks like this came from copy-pasting @AllenBW's use of backticks for the Migration Scheduled getText call. I'll fix them. Good catch!